src/pages/RFOX/components/Stake/StakeInput.tsx

```diff
@@ -0,0 +1,138 @@
+import { Button, CardFooter, Collapse, Stack } from '@chakra-ui/react'
+import { foxAssetId } from '@shapeshiftoss/caip'
+import { useCallback, useMemo, useState } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { useHistory } from 'react-router'
+import { Amount } from 'components/Amount/Amount'
+import { TradeAssetSelect } from 'components/AssetSelection/AssetSelection'
+import { FormDivider } from 'components/FormDivider'
+import { TradeAssetInput } from 'components/MultiHopTrade/components/TradeAssetInput'
+import { Row } from 'components/Row/Row'
+import { SlideTransition } from 'components/SlideTransition'
+import { WarningAcknowledgement } from 'components/WarningAcknowledgement/WarningAcknowledgement'
+import { selectAssetById } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+import { AddressSelection } from '../AddressSelection'
+import { StakeSummary } from './components/StakeSummary'
+import { StakeRoutePaths, type StakeRouteProps } from './types'
+
+const formControlProps = {
+  borderRadius: 0,
+  background: 'transparent',
+  borderWidth: 0,
+  paddingBottom: 0,
+  paddingTop: 0,
+}
+
+export const StakeInput: React.FC<StakeRouteProps> = ({ headerComponent }) => {
+  const translate = useTranslate()
+  const history = useHistory()
+  const asset = useAppSelector(state => selectAssetById(state, foxAssetId))
+  const [showWarning, setShowWarning] = useState(false)
+  const percentOptions = useMemo(() => [1], [])
+  const [cryptoAmount, setCryptoAmount] = useState('')
+  const [fiatAmount, setFiatAmount] = useState('')
+
+  const handleAccountIdChange = useCallback(() => {}, [])
+
+  const hasEnterValue = useMemo(() => !!fiatAmount || !!cryptoAmount, [cryptoAmount, fiatAmount])
+
+  const handleChange = useCallback((value: string, isFiat?: boolean) => {
```

Comment:
Doesn't need to be added yet but can we add some TODO re: adding a `toggleIsFiat` so we don't forget to?